### DISCUSSION TITLE
Table of contents numbering goes 1, 10, 2?

### DISCUSSION THREAD

**Discussion by MelissaPowerON** [requarks/wiki#6084] [Help / Questions]
Hello, 

How do I get my TOC in proper numerical order? It is currently forcing the number system to go 1., 10., 2., 3. etc. 

I am working with compliance documents that need to be in numerical order and need to have their document number in the TOC. For example, IMS-1, IMS-2, and so on, but the wiki keeps putting number 10 after number 1.

**Comment by NGPixel** [comment]
You mean the left navigation? The table of contents is not sorted and items are listed in the order they are on the page.

**Reply by MelissaPowerON** [reply]
Yes! Sorry, the left navigation is what I mean. I did see that there are different ways to organize the navigation, do you know if I can switch from one to the other after I have pages created? I am currently set to site tree, but I think static might be what I need. I just don't want to lose pages I already have.

**Reply by NGPixel** [reply]
You can switch navigation mode at any time. It's not tied to any content.

**Reply by MelissaPowerON** [reply]
Awesome! Thank you!
